src/PlatformInstaller.ClickOnce/PlatformInstaller.ClickOnce.csproj

```diff
@@ -80,7 +80,7 @@
       <ProdDeployManifest>PlatformInstaller.application</ProdDeployManifest>
       <TestDeployManifest>Test-PlatformInstaller.application</TestDeployManifest>
       <ProdDeployUrl>https://s3.amazonaws.com/particular.downloads/PlatformInstaller/</ProdDeployUrl>
-      <TestDeployUrl>https://particulartest.blob.core.windows.net/platforminstaller/</TestDeployUrl>
+      <TestDeployUrl>https://platforminstallerstaging.blob.core.windows.net/platforminstaller/Test-ParticularPlatform.exe</TestDeployUrl>
```

Comment:
Should the actual exe be in there? (the Prod URL doesn't have it?)

Reply:
Curse, you are right. I'll fix it.